tools/python_vardict_interval_split.cwl

```diff
@@ -29,7 +30,7 @@ arguments:
                 f = 0
                 if i not in intvl_set:
                     intvl_set[i] = []
-                data = cur_intvl.rstrip("\n").split("\t")
+                data = cur_intvl.rstrip(chr(10)).split(chr(9))
```

Comment:
should we comment what the `chr` does for the uncool kids?

Reply:
Can add